packages/react-server/core/FramebackController.js

```diff
@@ -228,40 +229,19 @@ class FramebackController extends EventEmitter {
 
 	createFrame(url){
```

Comment:
This is probably easier to look at with a split diff rather than unified.  The entire body of this method changed.